custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -219,6 +226,7 @@ def _load_via_predictors(self, model_artifact_file):
         return model
 
     def _find_predictor_to_use(self):
+        # TODO: RAPTOR-4014
```

Comment:
this comment could be fleshed out a little lol

Reply:
never